api/services/namespace_test.go

```diff
@@ -1075,78 +1512,223 @@ func TestEditNamespaceUser(t *testing.T) {
+			TenantID:      "a736a52b-5777-4f92-b0b8-e359bf484713",
+			UserID:        "activeMemberID",
+			MemberID:      "passiveMemberID",
 			MemberNewRole: guard.RoleObserver,
 			RequiredMocks: func() {
+				namespaceActiveOwner := &models.Namespace{
+					Name:     "group1",
+					Owner:    "activeMemberID",
+					TenantID: "a736a52b-5777-4f92-b0b8-e359bf484713",
+					Members: []models.Member{
+						{ID: "activeMemberID", Role: guard.RoleOwner},
+					},
+				}
+
+				activeMember := &models.User{
+					UserData: models.UserData{
+						Name:     "activeMemberName",
+						Username: "activeMemberUsername",
+					},
+					ID: "activeMemberID",
+				}
+
+				passiveMember := &models.User{
+					UserData: models.UserData{
+						Name:     "passiveMemberName",
+						Username: "passiveMemberUsername",
+					},
+					ID: "passiveMemberID",
+				}
+
+				mock.On("NamespaceGet", ctx, namespaceActiveOwner.TenantID).Return(namespaceActiveOwner, nil).Once()
+
+				mock.On("UserGetByID", ctx, activeMember.ID, false).Return(activeMember, 0, nil).Once()
+				mock.On("UserGetByID", ctx, passiveMember.ID, false).Return(passiveMember, 0, nil).Once()
+			},
+			Expected: NewErrNamespaceMemberNotFound("passiveMemberID", nil),
+		},
+		{
+			description:   "EditNamespaceUser fails when could not find passive member inside namespace",
+			TenantID:      "a736a52b-5777-4f92-b0b8-e359bf484713",
+			UserID:        "activeMemberID",
+			MemberID:      "passiveMemberID",
+			MemberNewRole: guard.RoleObserver,
+			RequiredMocks: func() {
+				namespaceActiveOwner := &models.Namespace{
+					Name:     "group1",
+					Owner:    "activeMemberID",
+					TenantID: "a736a52b-5777-4f92-b0b8-e359bf484713",
+					Members: []models.Member{
+						{ID: "activeMemberID", Role: guard.RoleOwner},
+					},
+				}
+
+				activeMember := &models.User{
+					UserData: models.UserData{
+						Name:     "activeMemberName",
+						Username: "activeMemberUsername",
+					},
+					ID: "activeMemberID",
+				}
+
+				passiveMember := &models.User{
+					UserData: models.UserData{
+						Name:     "passiveMemberName",
+						Username: "passiveMemberUsername",
+					},
+					ID: "passiveMemberID",
+				}
+
 				mock.On("NamespaceGet", ctx, namespaceActiveOwner.TenantID).Return(namespaceActiveOwner, nil).Once()
 
 				mock.On("UserGetByID", ctx, activeMember.ID, false).Return(activeMember, 0, nil).Once()
 				mock.On("UserGetByID", ctx, passiveMember.ID, false).Return(passiveMember, 0, nil).Once()
 			},
-			Expected: NewErrNamespaceMemberNotFound(passiveMember.ID, nil),
+			Expected: NewErrNamespaceMemberNotFound("passiveMemberID", nil),
 		},
 		{
-			Name:          "EditNamespaceUser fails when could not find active member inside namespace",
-			TenantID:      namespacePassiveObserver.TenantID,
-			UserID:        activeMember.ID,
-			MemberID:      passiveMember.ID,
+			description:   "EditNamespaceUser fails when could not find active member inside namespace",
```

Comment:
Should `description` be capitalized like the other fields?

Reply:
I think you could check to see if there is this inconsistency somewhere else.

Reply:
fixed in all cases